app/controllers/home_controller.rb

```diff
@@ -0,0 +1,6 @@
+class HomeController < ApplicationController
+
+  def dashboard
```

Comment:
I made a card for it and I will get it on my next PR if you don't mind. I need to change a few other calls to make sure it doesnt break anything.